services-camel/src/main/webapp/WEB-INF/service-context.xml

```diff
@@ -219,6 +219,35 @@
     <bean id="fulltextProcessor" class="edu.unc.lib.dl.services.camel.fulltext.FulltextProcessor">
         <constructor-arg value="${cdr.enhancement.path.fulltext}" />
     </bean>
+
+    <bean id="binaryInfoProcessor" class="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyedMsgProcessor">
```

Comment:
update name of the bean to match, the application wouldn't start for me until i did so 
```suggestion
    <bean id="destroyedMsgProcessor" class="edu.unc.lib.dl.services.camel.destroyDerivatives.DestroyedMsgProcessor">
```